integrations/wiz/main.py

```diff
@@ -26,31 +27,53 @@ def init_client() -> WizClient:
     )
 
 
-@ocean.on_resync()
-async def on_resync(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+@ocean.on_resync(kind=ObjectKind.PROJECT)
+async def on_resync_projects(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    """Resync projects from Wiz."""
+    wiz_client = init_client()
+    async for projects in wiz_client.get_projects():
+        logger.info(f"Received {len(projects)} projects")
+        yield projects
+
+
+@ocean.on_resync(kind=ObjectKind.ISSUE)
+async def on_resync_issues(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    logger.info(f"Resyncing {kind} kind")
+    wiz_client = init_client()
+
+    max_pages = cast(IssueResourceConfig, event.resource_config).selector.max_pages
+    status_list = cast(IssueResourceConfig, event.resource_config).selector.status_list
```

Comment:
I would change this to something like

selector = cast(IssueResourceConfig, event.resource_config).selector

then you can access it directly in the async for as selector.max_pages and selector.statuses